src/main/java/be/seeseemelk/mockbukkit/block/BlockMock.java

```diff
@@ -171,22 +176,49 @@ public void assertType(@NotNull Material material)
 	@Override
 	public byte getLightLevel()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return lightLevel;
+	}
+
+	/**
+	 * Sets the light level for this block.
+	 *
+	 * @param lightLevel Value between 0 and 15.
+	 */
+	public void setLightLevel(@Range(from=0, to=15) byte lightLevel)
+	{
+		this.lightLevel = lightLevel;
 	}
 
 	@Override
 	public byte getLightFromSky()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return lightFromSky;
+	}
+
+	/**
+	 * Sets the light level received from sky.
+	 *
+	 * @param lightFromSky Value between 0 and 15.
+	 */
+	public void setLightFromSky(@Range(from=0, to=15) byte lightFromSky)
+	{
+		this.lightFromSky = lightFromSky;
```

Comment:
Be more stringent with the input; have a precondition check for this and `setLightFromBlocks`